.drone.yml

```diff
@@ -0,0 +1,42 @@
+pipeline:
+   node 9:
+      image: bitgosdk/ubuntu_nvm:18.10
```

Comment:
Where does this image come from? Who built it? What if it needs to be changed, how can that be done?